docs/toc.yml

```diff
@@ -64,6 +64,9 @@ items:
     - name: Support older Microsoft webviews and Office versions
       href: develop/support-ie-11.md
       displayName: Excel, Word, OneNote, Project, PowerPoint, Outlook
+    - name: Managing both a unified manifest and an XML manifest version of your Office Add-in
```

Comment:
```suggestion
    - name: Manage add-in using unified and XML manifests
```

Reply:
I don't think the shorter version gets across the crucial idea. I realize that this is a verbose title, but every alternative that I've tried is just as bad.

Reply:
```suggestion
    - name: Manage both a unified manifest and an XML manifest version of add-in
```